pfio/v2/s3.py

```diff
@@ -193,11 +194,10 @@ def _flush(self):
 
         data = self.buf.getvalue()
         if 'b' in self.mode:
-            md5 = hashlib.md5(data).hexdigest()
+            md5 = base64.b64encode(hashlib.md5(data).digest()).decode()
         else:
-            md5 = hashlib.md5(data.encode()).hexdigest()
+            md5 = base64.b64encode(hashlib.md5(data.encode()).digest()).decode()
```

Comment:
This line yields CI error: `pfio/v2/s3.py:199:80: E501 line too long (80 > 79 characters)` by flake8.
Please add `# NOQA` or newline.

Reply:
Thanks! I solved the linting issues and now the CI is passing.